.github/ISSUE_TEMPLATE/icon_request.yml

```
@@ -0,0 +1,94 @@
+name: Icon request
+description: Request a new icon for Simple Icons
+title: "Request: "
+labels: [new icon]
+
+body:
+  - type: markdown
+    attributes:
+      value: >-
+        We won't add non-brand icons or anything related to illegal services. If in
+        doubt, open an issue and we'll have a look. For more details see the
+        [Contributing Guidelines][contributing-guidelines].
+
+
+        Before opening a new issue, make sure it isn't covered by an existing issue.
+        Please search for [issues with the `new icon` label][new-icon-issues]
+        (including closed issues) before you continue. If you find one for the brand
+        you're requesting then leave a comment on it or add a reaction.
+
+
+        [contributing-guidelines]:
+        https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md
+
+        [new-icon-issues]:
+        https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22new+icon%22+is%3Aopen
+
+  - type: input
+    attributes:
+      label: Brand Name
+      placeholder: "Example: Simple Icons"
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Website
+      description: >-
+        For non-web brands you can add a relevant link or simply put "None" if you don't
+        think there's a website.
+      placeholder: "Example: https://simpleicons.org"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Popularity Metric
+      description: >-
+        Provide either an [Alexa rank][alexa-site-info] or the number of GitHub stars.
+        If these are not available or too low you can still open the issue. Please
+        provide any information regarding the brand's popularity you think is relevant
+        below.
+
+
+        [alexa-site-info]: https://www.alexa.com/siteinfo
+      placeholder: "Example: 249,089"
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Official resources for icon and color
+      description: >-
+        Media kits, brand guidelines, SVG files, etc. We strongly encourage you to do
+        some research here as it will help implement your request faster, although you
+        can set this to none if you are unable to find any resources.
```

I'm personally not a fan of the wording "it will help implement your request faster" as it suggest we will always implement suggestions we receive (which is not an expectation we should set). From my side, it would be okay to remove the second sentence, but I'm open to keeping it if you have suggestions for wording that don't suggest we _will_ implement it.

And again, I think we should aim for shorter sentences.

```suggestion
        Media kits, brand guidelines, SVG files, etc. You can set
        this to none if you are unable to find any resources.
```

---

I think you're right, we should definitely avoid that expectation.